examples/slu/slurp/data_utils/prepare_slurp.py

```diff
@@ -0,0 +1,154 @@
+# ! /usr/bin/python
+# Copyright (c) 2022, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Code adapted from https://github.com/speechbrain/speechbrain/blob/develop/recipes/SLURP/prepare.py
+
```

Comment:
Hmm why is this script here and not in the dataset folder inside scripts directory? We have not come up with clean way of doing recipes yet but this is fine for just SLURP.